addons/binding/org.openhab.binding.pentair/src/main/java/org/openhab/binding/pentair/PentairBindingConstants.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.pentair;
+
+import java.util.Set;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+import com.google.common.collect.ImmutableSet;
+
+/**
+ * The {@link PentairBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Jeff James - Initial contribution
+ */
+public class PentairBindingConstants {
+
+    public static final String BINDING_ID = "pentair";
+
+    // List of Bridge Types
+    public static final String IP_BRIDGE = "ip_bridge";
+    public static final String SERIAL_BRIDGE = "serial_bridge";
+
+    // List of all Device Types
+    public static final String EASYTOUCH = "easytouch";
+    public static final String INTELLIFLO = "intelliflo";
+    public static final String INTELLICHLOR = "intellichlor";
+
+    // List of all Bridge Thing Type UIDs
+    public final static ThingTypeUID IP_BRIDGE_THING_TYPE = new ThingTypeUID(BINDING_ID, IP_BRIDGE);
```

Comment:
Non-preferred order of modifiers (It was like this in a template, which has been changed recently) correct order is `public static final` See also: http://stackoverflow.com/questions/16731240/what-is-a-reasonable-order-of-java-modifiers-abstract-final-public-static-e#16732059